src/a11y/index.js

```diff
@@ -1,7 +1,7 @@
 export function getAriaLabel(label) {
   if (!label) return undefined
 
-  if (label && label.length && label[0] === '#') {
+  if (label && label[0] === '#') {
     return { 'aria-labelledby': label.replace(/#/g, '') }
```

Comment:
Might be useful to leave a comment here explaining why we're replacing the hash...

Reply:
Check if that's good enough

Reply:
> // If input label reference ids (hash), use a labelledby without hashes

Yes, but why? Why do we need to replace the hash - that's the question.

Reply:
Because of you write "hello world" you are providing the actual label as a text.
If you write "#hello #world" as a label you reference two label by id, hello and world. The aria-labelledby reference these without hash.

Reply:
Do you want a refence to https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute#Examples or similar?

Reply:
> It's explained in the documentation for the label-prop.

Exactly! The "what" is obvious while reading the code but the "why" is not. A comment that points out the why (it can be as simple as pointing out the readme#label prop) helps answer the "why".

I generally prefer the code to be self-documenting but sometimes, things are not obvious or clear just by reading the code. This is one such case IMHO. To the next person, "why is this hash replacement happening" is a question that is not obvious; which is why a comment is mighty helpful!

Makes sense?